### DISCUSSION TITLE
[预览] v2.9.0

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4835] [分享 / Share]
✨新增
- `简化直播间` 支持屏蔽推荐直播间. (#4787)
- 新增功能 `删除直播马赛克遮罩` (#4634, PR #4814)
> 删除观看直播时某些分区的马赛克遮罩.

- `启用视频截图` 截出来的图支持直接复制. (此功能需要 Firefox 127 版本以上) (#4806)
- `图片批量导出`, `下载视频` 支持更多变量, 详情可在更新组件后查看设置中的说明: (#3852)
  - 动态 ID, 用户 ID, 动态发布时间, 被转发动态相关数据
  - 专栏 cv 号, 专栏发布时间
  - (仅对批量视频下载 (分P / 合集) 有效) up 主名称, up 主 ID, 视频发布时间
- `自定义顶栏` 的稍后再看和历史面板现在始终显示 "已观看" 状态. (#4346)
- `自定义顶栏` 的稍后再看, 收藏和历史面板优化了分 P 数和观看进度的展示, 详见[此处](https://github.com/the1812/Bilibili-Evolved/discussions/1866#discussioncomment-10075203). (#1866)

🗑️废弃
- 删除 `下载视频` 的 Toast 输出方式.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.0-preview'>[预览] v2.9.0</a>.</em>

**Comment by Tinhone** [comment]
欸，话说为啥删除了 Toast 输出方式

**Reply by the1812** [reply]
StreamSaver 是更好的替代品, Toast 至今仍然不稳定
